config/samples/osbuilder_v1alpha1_osbuild.yaml

```diff
@@ -3,4 +3,16 @@ kind: OSBuild
 metadata:
   name: osbuild-sample
 spec:
-  # TODO(user): Add fields here
+  details:
+    distribution: fedora-35
```

Comment:
s/fedora-35/rhel-90/

Fedora does not support `edge-container` image type, just `iot-container`.